serde_codegen/src/ser.rs

```diff
@@ -353,26 +278,23 @@ fn serialize_variant(
                 &variant.fields[0],
             );
 
-            quote_arm!(cx,
-                $type_ident::$variant_ident(ref __simple_value) => $block
-            )
+            quote! {
+                // The braces are unnecessary but quasi used to put them in. We
```

Comment:
If I recall correctly, quote_block required the extra braces, so shouldn't be a problem to remove them later.